blueprints/ember-cli-moment-shim/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = {
+  normalizeEntityName: function() {
+    // this prevents an error when the entityName is
+    // not specified (since that doesn't actually matter
+    // to us
+  },
+
+  afterInstall: function() {
+    return this.addBowerPackageToProject('ember-cli-moment-shim');
```

Comment:
I'm thinking we can avoid all of this by using the npm dep instead of the bower dep and pick the files into the vendor tree.

Eventually, I'll consume the moment 2.10.x ES6 modules